src/accelerate/test_utils/testing.py

```diff
@@ -163,6 +180,16 @@ def require_single_xpu(test_case):
     return unittest.skipUnless(torch.xpu.device_count() == 1, "test requires a XPU")(test_case)
 
 
+def require_multi_accelerator(test_case):
```

Comment:
```suggestion
def require_distributed(test_case):
```

Reply:
In order to correspond to `require_single_device`, I tend to rename `rquire_multi_accelerator` to `require_multi_device`